managed/src/main/java/com/yugabyte/yw/controllers/CloudProviderController.java

```diff
@@ -55,6 +56,12 @@
 import static com.yugabyte.yw.models.helpers.CommonUtils.DEFAULT_YB_HOME_DIR;
 
 public class CloudProviderController extends AuthenticatedController {
+  private final Config config;
+  @Inject
```

Comment:
Insert empty string before @Inject. Each function should be separated from fields and other functions by an empty line.

Reply:
done